src/main/java/uk/gov/hmcts/probate/model/ccd/raw/request/CaseDataParent.java

```diff
@@ -3,18 +3,30 @@
 import lombok.Data;
 import lombok.experimental.SuperBuilder;
 import lombok.extern.jackson.Jacksonized;
+import uk.gov.hmcts.probate.model.ccd.raw.AdditionalExecutorTrustCorp;
+import uk.gov.hmcts.probate.model.ccd.raw.CollectionMember;
+
+import java.time.LocalDate;
+import java.util.List;
 
 @Jacksonized
 @SuperBuilder
 @Data
 public class CaseDataParent {
 
     protected String registrySequenceNumber;
-    protected String dispenseWithNotice;
-    protected String titleAndClearingType;
```

Comment:
did we definitely want to remove these 2? Just checking not merged out accidentally